bugsnag-android-core/src/main/java/com/bugsnag/android/ThreadState.kt

```diff
@@ -78,6 +81,7 @@ internal class ThreadState @Suppress("LongParameterList") constructor(
         exc: Throwable?,
         isUnhandled: Boolean,
         maxThreadCount: Int,
+        maxTime: Long,
```

Comment:
naming: this could be clearer

```suggestion
        stacktraceCollectionTimeout: Long,
```